test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -43,7 +52,14 @@ const (
 	configFile        = "config/performance-config.yaml"
 	createNodesOpcode = "createNodes"
 	createPodsOpcode  = "createPods"
+	churnOpcode       = "churn"
 	barrierOpcode     = "barrier"
+
+	// Two modes supported in "churn" operator.
```

Comment:
nit: add empty line, as this comment refers to more than one constant

Reply:
done.